### DISCUSSION TITLE
Skipping model load at the start

### DISCUSSION THREAD

**Discussion by Sreerag-ibtl** [lllyasviel/Fooocus#2532] [Q&A] [answered]
Can we skip model loading at the start? I didn't see any cmd flag matching it. As a side note, I plan to use Fooocus as an API rather than UI. Loading models in the start will occupy GPU making other services in the container to throw OOM errors.

**Comment by mashb1t** [comment, marked as answer]
This is only possible when changing the code and commenting / removing these lines: https://github.com/lllyasviel/Fooocus/blob/main/modules/default_pipeline.py#L254-L258
Wouldn't the otehr services then go OOM when starting the image generation the container anyways?

**Reply by Sreerag-ibtl** [reply]
I have managed to get the desired effect by setting default models to None in config.txt and specifying the model names in the input request instead. So far, it's working as expected. Regarding OOM, it has been reduced to a considerable degree. There's still room for improvement.
